Gems/PhysX/Code/Editor/Source/Components/EditorSystemComponent.cpp

```diff
@@ -138,6 +138,9 @@ namespace PhysX
                     if (auto retrievedMaterialLibrary = RetrieveDefaultMaterialLibrary())
                     {
                         physxSystem->UpdateMaterialLibrary(retrievedMaterialLibrary.value());
+
+                        // After setting the default material library, save the physx configuration.
+                        physxSystem->GetSettingsRegistryManager().SaveSystemConfiguration(physxSystem->GetPhysXConfiguration(), {});
```

Comment:
question: Should there be a callback provided to handle a save settings failure case (handling can just be logging the result)?

Reply:
This case is going to be run during loading only if there is no physics material assigned and a default needs to be set, this is equivalent to `SystemComponent::ActivatePhysXSystem()` where callbacks are not provided either, I followed the same pattern: 
````
registryManager.SaveSystemConfiguration(defaultConfig, {});
````

Let me know if you think we should add callbacks to all of those.

Reply:
sounds good to me!

Reply:
done